core/src/components/picker/test/basic/picker.e2e.ts

```diff
@@ -9,11 +9,11 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     test('inline pickers should not have visual regression', async ({ page }) => {
       await page.goto(`/src/components/picker/test/basic`, config);
 
-      const fullStack = page.locator('#inline button[data-value="full-stack"]');
-      const onion = page.locator('#inline button[data-value="onion"]');
+      const fullStack = page.locator('#inline ion-picker-column-option[data-test-value="full-stack"]');
+      const onion = page.locator('#inline ion-picker-column-option[data-test-value="onion"]');
 
-      await expect(fullStack).toHaveClass(/picker-item-active/);
-      await expect(onion).toHaveClass(/picker-item-active/);
+      await expect(fullStack).toHaveClass(/option-active/);
```

Comment:
It seems to have taken two tries to pass. I wonder if there's a way to minimize the retries. This isn't a blocker but can be worth a conversation.